### PR TITLE
Proposed Appointment Write Documentation Updates

### PR DESCRIPTION
**Pull request by exopsRamba**
Description
----
Adding documentation for creating new Appointments with a status of 'proposed'.

Local Verification - Before
----
Implementation Notes - Before
Example - Before

Local Verification - After
---
Implementation Notes - After
Booked Example - After
Proposed Example - After
Requested Period Field

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/scheduling/appointment.md [hunk 1]
```diff
@@ -312,7 +318,39 @@ _Implementation Notes_
...
-### Example
+### Example - Proposed Appointment
+
+#### Request
+
+    POST https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Appointment
+
+#### Body
+
+  <%= json(:r4_appointment_request_create) %>
```
Comment (MadhurRaj) [inline content/millennium/r4/scheduling/appointment.md:329]: let's change this to r4_proposed_appointment_create?

File: lib/resources/r4/appointment.yaml [hunk 2]
```diff
@@ -14,37 +14,86 @@ fields:
...
+      "status": "booked"
+    }
+  note: When creating a new appointment, the status must be `booked` or `proposed`.
+
+- name: serviceType
+  required: 'No'
+  type: CodeableConcept
+  description: The specific service that is to be performed during this appointment.
+  action:
+  - read
```
Comment (MadhurRaj) [inline lib/resources/r4/appointment.yaml:33]: read? there shouldn't be a read table...

File: lib/resources/r4/appointment.yaml [hunk 3]
```diff
@@ -14,37 +14,86 @@ fields:
...
+              "display": "Surgery-general",
+              "userSelected": false
+            }
+          ],
+          "text": "Surgery"
         }
       ]
     }
-  note: At most one slot can be populated.
+  url: http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.serviceType
```
Comment (MadhurRaj) [inline lib/resources/r4/appointment.yaml:58]: https

File: lib/resources/r4/appointment.yaml [hunk 4]
```diff
@@ -14,37 +14,86 @@ fields:
   note: resourceType must be Appointment
   url: http://hl7.org/fhir/r4/json.html#resources
```
Comment (MadhurRaj) [inline lib/resources/r4/appointment.yaml:15]: https

File: lib/resources/r4/appointment.yaml [hunk 5]
```diff
@@ -94,48 +143,10 @@ fields:
     }
   url: http://hl7.org/fhir/r4/appointment-definitions.html#Appointment.comment
```
Comment (MadhurRaj) [inline lib/resources/r4/appointment.yaml:144]: https

File: lib/resources/r4/appointment.yaml [hunk 6]
```diff
@@ -155,16 +166,25 @@ fields:
     }
   url: http://hl7.org/fhir/R4/appointment-definitions.html#Appointment.participant.type
```
Comment (MadhurRaj) [inline lib/resources/r4/appointment.yaml:167]: https

File: lib/resources/r4/appointment.yaml [hunk 7]
```diff
@@ -155,16 +166,25 @@ fields:
...
+  example: |
+    {
+      "requestedPeriod": [
+        {
+          "start": "2015-02-07T13:28:17-05:00",
+          "end": "2016-02-07T13:28:17-05:00"
+        }
+      ]
+    }
+  url: http://hl7.org/fhir/appointment-definitions.html#Appointment.requestedPeriod
```
Comment (MadhurRaj) [inline lib/resources/r4/appointment.yaml:190]: https

File: content/millennium/r4/scheduling/appointment.md (hunk 1 above)
Comment (MaxPhilips) [reply content/millennium/r4/scheduling/appointment.md]: https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-1d7df8ceb9f0ae8471323516e037c662R329
https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-1a83ec9f4ba450cba6a16d8bcda1890cR366

File: lib/resources/r4/appointment.yaml (hunk 3 above)
Comment (MaxPhilips) [reply lib/resources/r4/appointment.yaml]: https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-fbb215a73c336ca1ac34d91de2fb411eR58

File: lib/resources/r4/appointment.yaml (hunk 4 above)
Comment (MaxPhilips) [reply lib/resources/r4/appointment.yaml]: https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-fbb215a73c336ca1ac34d91de2fb411eR15

File: lib/resources/r4/appointment.yaml (hunk 5 above)
Comment (MaxPhilips) [reply lib/resources/r4/appointment.yaml]: https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-fbb215a73c336ca1ac34d91de2fb411eR144

File: lib/resources/r4/appointment.yaml (hunk 6 above)
Comment (MaxPhilips) [reply lib/resources/r4/appointment.yaml]: https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-fbb215a73c336ca1ac34d91de2fb411eR167

File: lib/resources/r4/appointment.yaml (hunk 7 above)
Comment (MaxPhilips) [reply lib/resources/r4/appointment.yaml]: https://github.com/cerner/fhir.cerner.com/pull/392/commits/53154ecf808c864284463c039e4ca41cd2b708cc#diff-fbb215a73c336ca1ac34d91de2fb411eR190

File: lib/resources/r4/appointment.yaml (hunk 2 above)
Comment (MaxPhilips) [reply lib/resources/r4/appointment.yaml]: https://github.com/cerner/fhir.cerner.com/pull/392/commits/741818815c57461d0bc7048e1446e6114838fedc

**Comment by MaxPhilips** [comment]
verified on live

Screen Shot 2020-06-05 at 10 56 20 AM

**Pull request merged**